net-irc/anope/anope-2.0.7.ebuild

```diff
@@ -0,0 +1,154 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit cmake
+
+DESCRIPTION="Anope IRC Services"
+HOMEPAGE="https://anope.org"
+SRC_URI="https://github.com/anope/anope/releases/download/${PV}/${P}-source.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="chanstats cs-fantasy-stats cs-fantasy-top gnutls irc2sql ldap
+	ldap-auth ldap-oper mysql pcre posix sql-auth sql-log sql-oper
+	sqlite ssl tre"
+
+REQUIRED_USE="
+	chanstats? ( mysql )
+	cs-fantasy-stats? ( chanstats mysql )
+	cs-fantasy-top? ( chanstats mysql )
+	irc2sql? ( mysql )
+	ldap-auth? ( ldap )
+	ldap-oper? ( ldap )
+	sql-auth? ( || ( mysql sqlite ) )
+	sql-log? ( mysql )
+	sql-oper? ( mysql )"
+
+BDEPEND="sys-devel/gettext"
+DEPEND="${BDEPEND}
+	acct-group/anope
+	acct-user/anope
+	mysql? ( dev-db/mysql-connector-c:0= )
+	ssl? ( dev-libs/openssl:0= )
+	gnutls? ( net-libs/gnutls:0= dev-libs/libgcrypt:0= )
+	ldap? ( net-nds/openldap )
+	pcre? ( dev-libs/libpcre )
+	sqlite? ( dev-db/sqlite:3 )
+	tre? ( dev-libs/tre )
+	virtual/libintl
+"
+RDEPEND="${DEPEND}"
+
+PATCHES=( "${FILESDIR}/pid-patch.patch" )
+
+S="${WORKDIR}/${P}-source"
+
+pkg_pretend() {
+	local replaced_version
+	for replaced_version in ${REPLACING_VERSIONS} ; do
+		if ver_test ${replaced_version} -lt 1.8.7 && [[ -f ${ROOT}/opt/anope/data/nick.db ]]; then
```

Comment:
I'll investigate nick.db but I agree, we can probably just leave it. A lot has changed since then. 

I think there may have been other DB migrations too.